### ISSUE TITLE
Request for Tutorial or further Documentation/Example

### ISSUE THREAD

**Issue by kitaedesigns** [0beqz/realism-effects#25]
Difficult to setup and integrate into an existing three.js project. Setting up the Offscreen Canvas to use elsewhere is not explained. Also doesn't seem to work at all on mobile because of this?

**Comment by 0beqz** [comment]
Agreed, I think the best way is to add many codesandbox examples once the new version is out. Especially when it's integrated into r3f-postprocessing.
